### PR TITLE
add installation.md

### PR DESCRIPTION
**Pull request by liangnuaa**
add installation.md for https://github.com/jxnl/instructor-js/issues/36

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
-!!! warning "Page under construction"
+Installation is as simple as:
 
-    This page is under construction. Please check back later. Consider contributing to this page by opening a PR! 
\ No newline at end of file
+```bash
+npm install instructor-js
```
Comment (jxnl) [inline docs/installation.md:4]: not true yet, as we cannot get `instructor` on npm, can you keep it a warning

Comment (jxnl) [reply docs/installation.md:4]: might have to suggest installing from github

File: docs/contributing.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-We would love for you to contribute to `Instructor`.
+We would love for you to contribute to `Instructor-js`.
 
-## [Evals](https://github.com/jxnl/instructor/tree/main/tests/openai/evals)
+## [Evals](https://github.com/jxnl/instructor-js/tree/main/tests/openai/evals)
 
-We invite you to contribute evals in pytest as a way to monitor the quality of the openai models and the instructor library. To get started check out the [jxnl/instructor/tests/evals](https://github.com/jxnl/instructor/tree/main/tests/openai/evals) and contribute your own evals in the form of pytest tests. These evals will be run once a week and the results will be posted.
+We invite you to contribute evals in Jest as a way to monitor the quality of the openai models and the instructor library. To get started check out the [jxnl/instructor-js/tests/evals](https://github.com/jxnl/instructor-js/tree/main/tests/openai/evals) and contribute your own evals in the form of pytest tests. These evals will be run once a week and the results will be posted.
```
Comment (jxnl) [inline docs/contributing.md:5]: Bun not jest*

**Review by jxnl — CHANGES_REQUESTED** [review]

File: docs/installation.md (hunk 1 above)
Comment (liangnuaa) [reply docs/installation.md:6]: added

**Pull request merged**